x-pack/plugins/encrypted_saved_objects/index.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Root } from 'joi';
+import { Legacy, Server } from 'kibana';
+
+// @ts-ignore
+import { AuditLogger } from '../../server/lib/audit_logger';
+
+import { CONFIG_PREFIX, PLUGIN_ID, Plugin } from './server/plugin';
```

Comment:
__note:__ tried to follow the way promoted by the new platform migration guide - where we split new-platform-like plugin part into a dedicated file.